pkg/monitoring/metrics/virt-controller/migrationstats_collector.go

```diff
@@ -62,15 +62,15 @@ var (
 			Name: "kubevirt_vmi_migration_succeeded",
 			Help: "Indicates if the VMI migration succeeded.",
 		},
-		[]string{"vmi", "vmim"},
+		[]string{"vmi", "vmim", "namespace"},
```

Comment:
can a migration be made to a different namespace? if so, it is not clear the meaning of this namespace
maybe `source_namespace` / `target_namespace`?

Reply:
> can a migration be made to a different namespace?

Not that I'm aware of. AFAIK it's about migration to another host. @iholder101 Any chance you know?

Reply:
I didn't see anything in the docs